### DISCUSSION TITLE
Latest release 0.0.21

### DISCUSSION THREAD

**Discussion by bug-or-feature** [ig-python/ig-markets-api-python-library#302] [Announcements]
The latest release includes support for the official Lightstreamer Python client SDK, as IG have recently updated their server version to 7.3.3. The stream sample has been updated too. See

- https://pypi.org/project/lightstreamer-client-lib/1.0.3/
- https://github.com/ig-python/trading-ig/discussions/253
- https://labs.ig.com/lightstreamer-downloads
- https://sdk.lightstreamer.com/ls-python-client/1.0.3/api/index.html

**Comment by blobyNob** [comment]
great work. thanks for the efforts
